config/webpack.config.js

```diff
@@ -300,6 +300,13 @@ module.exports = function(webpackEnv) {
 
         // First, run the linter.
         // It's important to do this before Babel processes the JS.
+        {
```

Comment:
Why do you need two `webpack.config.js` files?
Remove it as a create-react-app file.